internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -975,6 +975,12 @@ components:
       required:
         - profile_id
       properties:
+        policy_id:
```

Comment:
I'm unsure if this belongs here, could add a separate 'Compliance' customisation.

Reply:
Yeah this is a fair point. I don't have strong feelings though